random_walk.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+np.random.seed(3456)
+# Simulating a random walk 1000 times
+
+all_walks = []
+for x in range(1000):
+    random_walk = [0]
+    for y in range(100):
+        step = random_walk[-1]
+        dice = np.random.randint(1, 7)
+        if dice <= 2:
```

Comment:
The logic how you generate the next step is not obvious. It seems you want to generate 3 states, in that case can it be:

`dice = np.random.randint(1, 3)`